app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -186,6 +186,10 @@ def destroy_vm(uuid)
       true
     end
 
+    def supports_vms_pagination?
+      true
```

Comment:
No, I will pin rbovirt in this PR once the PR there gets merged and a new version will be released.